spec/support/command_helpers.rb

```diff
@@ -1,17 +1,270 @@
 # frozen_string_literal: true
 
-module CommandHelpers
-  def command_output
-    tmp_stderr = Tempfile.create
+require "securerandom"
+require "pty"
+require "expect"
 
-    allow_any_instance_of(Command::Base).to receive(:progress).and_return(tmp_stderr) # rubocop:disable RSpec/AnyInstance
+class SpawnedCommand
```

Comment:
It is not very good to have class in different file name. I guess either extract it to separate file with same name or put inside below module namespace.

Reply:
Done in 9b9282ab0ed2cdaf53e0822ef579409618e781d1.